### ISSUE TITLE
Needs verification / Preparing for production

### ISSUE THREAD

**Issue by daparker** [glotlabs/gdrive#129]
After following all of the steps to set up the Google project and obtain the OAuth credentials, I received a screen that says:

> ### Verification Status
> **Needs verification**
> 
> Because you're using one or more sensitive scopes, your app registration requires verification by Google. Please prepare your app to submit for verification.

And it has a **PREPARE FOR VERIFICATION** button. In a panel on the right-hand side, it says:

> ### Preparing for production
> 
> You've configured your app in a way that requires verification . To complete verification, you will need to provide:
> 
> An official link to your app's Privacy Policy
> A YouTube video showing how you plan to use the Google user data you get from scopes
> A written explanation telling Google why you need access to sensitive and/or restricted user data
> All your domains verified in Google Search Console

Is this tool still usable without going through their extensive process?

**Comment by Akutasan** [comment]
Yes, it is as you will most likely be the only person using the OAuth screen. Once you use the add command, you'll be presented with a link which you open and you can just go to "Advanced" and click on proceed (unsafe) to use it.
